### ISSUE TITLE
Fail to create a project. Error: end of central directory record signature not found

### ISSUE THREAD

**Issue by Mattkitkat** [microsoft/vscode-spring-initializr#191]
Hi, I tried following this guide: 

https://code.visualstudio.com/docs/java/java-spring-boot

And believe to have installed all the prerequisites however am running into this issue below when trying to initialize a spring project: 

![image](https://user-images.githubusercontent.com/9200710/142729003-3a4b9604-16ac-42a3-8faf-ba4211c8b393.png)

I believe someone else has encountered this issue but has reported it in a different repository:
https://github.com/microsoft/vscode-java-pack/issues/818

**Comment by TigerTurnedLion** [comment]
Same. looks like a problem with the downloaded zip package. The following message flashes before error: "Starting to unzip..."

Added this comment to both repos

**Comment by Eskibear** [comment]
We didn't release any new version recently.  I guess something was wrong when it tried to unzip the package. The zip package is downloaded from "https://start.spring.io", which can be possibly changed and not coverred by `extract-zip` lib used by this extension.

I'll investigate it later.

**Comment by Charradii** [comment]
I'm facing the same problem, it used to work but today i was about to create a new project and the error appeared

**Comment by Eskibear** [comment]
I did a quick investigation. The root cause is wrong `path` of the downloading URL, leading to a broken zip package.
`path` should be `/starter.zip` instead of `//starter.zip`.  Previously it was working well even with the double slashes. Maybe it's because of some recent updates in spring initializr service. I'll fix it ASAP.

```
# Downloading URL:
https://start.spring.io//starter.zip?type=maven-project&language=java&javaVersion=11&groupId=com.example&artifactId=demo&name=demo&packaging=jar&bootVersion=2.6.0&baseDir=demo&dependencies=

# Response:
{"timestamp":"2021-11-21T05:17:23.017+00:00","status":404,"error":"Not Found","message":"No message available","path":"//starter.zip"}
```

**Comment by Eskibear** [comment]
fixed in v0.7.1
